angrmanagement/ui/workspace.py

```diff
@@ -3,6 +3,7 @@
 import traceback
 
 from PySide2.QtCore import Qt, QSettings
+from string import ascii_uppercase
```

Comment:
@woadey I believe `string` is a built-in python library. Could you place this import next to `traceback` above, so that all imports of that kind are together.